ratelimits/limit_test.go

```diff
@@ -0,0 +1,46 @@
+package ratelimits
+
+import (
+	"testing"
+
+	"github.com/letsencrypt/boulder/test"
+)
+
+func Test_loadLimits(t *testing.T) {
+	_, err := loadLimits("test/defaults.yml")
+	test.AssertNotError(t, err, "should not error")
+
+	_, err = loadLimits("test/overrides.yml")
+	test.AssertNotError(t, err, "should not error")
+
+	_, err = loadLimits("")
+	test.AssertError(t, err, "should error")
```

Comment:
I don't believe this has been addressed; for example, it took me a distressingly long time to realize that "busted_override_limit.yml" was supposed to error because it's burst capacity is set to 0, not because there's anything wrong with its name.

Reply:
I've rewritten all of the test cases to be more descriptive.